src/Aspire.Dashboard/Resources/xlf/Login.fr.xlf

```diff
@@ -9,7 +9,7 @@
       </trans-unit>
       <trans-unit id="HelpScreenshotAltText">
         <source>Screenshot of example console output showing where to find the dashboard frontend token. A red box highlights the token at the end of a url. The token is the value of a query parameter named 't'.</source>
-        <target state="needs-review-translation">Capture d’écran de la sortie de console illustrant où trouver le jeton front-end du tableau de bord</target>
```

Comment:
I am not familiar with the loc process, but is it expected that this checkin changes back to English? @kvenkatrajan

Reply:
Thanks for catching this. Yes thats not right. Opened up an ICM ticket - CCed you on it.

Reply:
Please go ahead and reject this. As per our division loc expert - 

_This is normal, if the source string (English) is updated or is being detected as being updated.
The translations will be reverted to English until the new translations are back.
You can see this scenario by reviewing the attribute 'state' in the target node:
state = "needs-review-translation"
When the state is like the above, the build will produce the target resx files with the original English.
You could either accept the change or keep rejecting until the translation is back._